producers/mysql-producer/src/main/java/com/flipkart/aesop/runtime/producer/eventprocessor/impl/DeleteEventV2Processor.java

```diff
@@ -33,9 +38,18 @@ public void process(BinlogEventV4 event, OpenReplicationListener listener) throw
 		}
 		LOGGER.debug("Delete Event Received : " + event);
 		DeleteRowsEventV2 deleteRowsEvent = (DeleteRowsEventV2) event;
+		List<Row> rowList = deleteRowsEvent.getRows();
+		List<Pair<Row>> listOfPairs = new ArrayList<Pair<Row>>(rowList.size());
```

Comment:
If Java 7 is the source compatibility version, use diamond operator in all the RHS.